parcel_test.go

```diff
@@ -31,57 +33,106 @@ func getTestParcel() Parcel {
 // TestAddGetDelete проверяет добавление, получение и удаление посылки
 func TestAddGetDelete(t *testing.T) {
 	// prepare
-	db, err := // настройте подключение к БД
+	// настройте подключение к БД
+	db, err := sql.Open("sqlite", "tracker.db")
+	require.NoError(t, err)
+	defer db.Close()
+
 	store := NewParcelStore(db)
 	parcel := getTestParcel()
 
 	// add
 	// добавьте новую посылку в БД, убедитесь в отсутствии ошибки и наличии идентификатора
-
+	id, err := store.Add(parcel)
+	require.NoError(t, err)
+	assert.NotEqual(t, 0, id)
+	parcel.Number = id
 	// get
 	// получите только что добавленную посылку, убедитесь в отсутствии ошибки
+	newParcel, err := store.Get(id)
+	require.NoError(t, err)
 	// проверьте, что значения всех полей в полученном объекте совпадают со значениями полей в переменной parcel
+	assert.Equal(t, parcel.Number, newParcel.Number)
+	assert.Equal(t, parcel.Client, newParcel.Client)
+	assert.Equal(t, parcel.Status, newParcel.Status)
+	assert.Equal(t, parcel.Address, newParcel.Address)
+	assert.Equal(t, parcel.CreatedAt, newParcel.CreatedAt)
 
 	// delete
 	// удалите добавленную посылку, убедитесь в отсутствии ошибки
+	err = store.Delete(id)
+	require.NoError(t, err)
 	// проверьте, что посылку больше нельзя получить из БД
+	_, err = store.Get(id)
+	require.Equal(t, sql.ErrNoRows, err)
```

Comment:
Для сравнивания ошибок нужно использовать специализированную функцию
```suggestion
	require.ErrorIs(t, err, sql.ErrNoRows)
```